core/assets/jsons/global.json

```diff
@@ -33,17 +41,27 @@
       "maxspeed": 5.0,
       "startframe": 0,
       "walklimit": 4,
-      "texture": "enemy",
+      "texture":     	{
+        "left": "enemy",
+        "right": "enemy",
+        "up": "enemy",
+        "down": "enemy",
+        "walk-left": "enemy",
+        "walk-right": "enemy"
```

Comment:
What is the difference between "left" and "walk-left" and between "right" and "walk-right"?

Reply:
Was a difference originally, but deprecated now. Will remove it.